sources/engine/Stride.Graphics/Direct3D/SharpDXInterop.cs

```diff
@@ -71,13 +71,14 @@ public static object GetNativeRenderTargetView(Texture texture)
         /// <param name="device">The GraphicsDevice in use</param>
         /// <param name="dxTexture2D">The DX11 texture</param>
         /// <param name="takeOwnership">If false AddRef will be called on the texture, if true will not, effectively taking ownership</param>
+        /// <param name="toSRgb">Set the format to SRgb</param>
         /// <returns></returns>
-        public static Texture CreateTextureFromNative(GraphicsDevice device, object dxTexture2D, bool takeOwnership)
+        public static Texture CreateTextureFromNative(GraphicsDevice device, object dxTexture2D, bool takeOwnership, bool toSRgb = false)
```

Comment:
Can you rename everything as `isSrgb` like in the `InitializeFromImpl` parameter?
`toSrgb` sounds like it's going to transform something, rather than "interpreting as".

Reply:
good point, done!